assets_helper.py

```diff
@@ -28,7 +30,7 @@ def is_active(asset, at_time=None):
 
     if asset['is_enabled'] and asset['start_date'] and asset['end_date']:
         at = at_time or get_time()
-        return asset['start_date'] <= at <= asset['end_date']
+        return asset['start_date'] < at < asset['end_date']
```

Comment:
Actually we're changing this in a coming version. You're right it won't make much of a practical difference though. We're just changing it because we're detail obsessed developers. :)